js/app/test/json_component_blocks.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { test, expect } from "@gradio/tootils";
```

Comment:
The `json_component_blocks` demo does not exist so we should remove this test file.

Reply:
Ah sorry forgot to rename this.
This should be `json_component.spec.ts` instead of deleting.